initial-balances-generator-etherscan.js

```diff
@@ -76,6 +76,8 @@ var end = start + pageSize;
 			apiKey
 		}`
 		let resp = await axios.get(url)
+		console.assert(resp.data.status === "1", "Bad status in response: %s. Url: %s", resp.data.status, url)
```

Comment:
I see this is 0 if no records found, that may happen. In that case message is "No records found". So this PR needs to be amended.

Reply:
Addressed in force-push.